src/_xoConnection.js

```diff
@@ -137,14 +137,14 @@ class XoConnection extends Xo {
     }
     let backups = {};
```

Comment:
It should be a const.

Reply:
But it will be reassigned, so it cannot be a const. (l. 144)

Reply:
Why not mutate it?